### PR TITLE
[ Tenant-Operator ] Update tenant spec

### PR DESCRIPTION
**Pull request by hussnain612**
Updated feature in tenant spec

### CODE REVIEW

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-314-8dbd2df2`

File: content/sre/tenant-operator/customresources.md [hunk 1]
```diff
@@ -92,9 +92,23 @@ spec:
...
+        namespaces:
+          - alpha-dev
+          - alpha-build
   namespaceAnnotations:
-    openshift.io/node-selector: node-role.kubernetes.io/infra=
+    common:
+      openshift.io/node-selector: node-role.kubernetes.io/infra=
+    specific:
+      - annotations:
+          stakater.com/current-tenant: alpha
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/customresources.md]: please remove this annotation as it's not that meaningful. Secondly, this annotation is added by TO, so we don't need to mention it over here. Use some other meaningful annotation

File: content/sre/tenant-operator/customresources.md [hunk 2]
```diff
@@ -92,9 +92,23 @@ spec:
   - build
   - preview
   namespaceLabels:
-    app.kubernetes.io/managed-by: tenant-operator
+    common:
+      app.kubernetes.io/managed-by: tenant-operator
```
Comment (abdulhaseeb2) [inline content/sre/tenant-operator/customresources.md]: Use a meaningful label that makes sense. As this will guide the user that what they can add.

**Review by abdulhaseeb2 — CHANGES_REQUESTED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-314-42c6a7e7`

File: content/sre/tenant-operator/customresources.md (hunk 1 above)
Comment (hussnain612) [reply content/sre/tenant-operator/customresources.md]: Updated

File: content/sre/tenant-operator/customresources.md (hunk 2 above)
Comment (hussnain612) [reply content/sre/tenant-operator/customresources.md]: Updated

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-314-22b5ebf0`

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-314-7323f23c`

**Review by ShahidHussain-srti — DISMISSED** [review]

**Comment by stakater-user** [comment]
@hussnain612 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:SNAPSHOT-PR-314-808eed12`

**Review by abdulhaseeb2 — APPROVED** [review]

**Comment by stakater-user** [comment]
@abdulhaseeb2 Image is available for testing. `docker pull docker-stakater-nexus.apps.prod.clusters.stakater.io/stakater/stakaterclouddocs:v0.0.253`

**Pull request merged**